.travis.yml

```diff
@@ -0,0 +1,46 @@
+language: php
+sudo: false
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+cache:
+  directories:
+    - node_modules
+    - vendor
+    - $HOME/phpunit-bin
+
+addons:
+  apt:
+    packages:
+      - libxml2-utils
+
+language:
+  - php
+  - node_js
+
+node_js:
+  - "6"
+
+php:
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+
+env:
+  - WP_VERSION=latest WP_MULTISITE=0
+  - WP_VERSION=trunk WP_MULTISITE=1
```

Comment:
Hi @ThierryA, is the plan to only support the latest version of WordPress?

Reply:
Yes